snakes_cafe.py

```diff
@@ -1,32 +1,24 @@
-appetizers = ['Appetizers:\n','----------', 'Wings', 'Cookies', 'Spring Rolls\n']
-entrees = ['Entrees:', '----------','Salmon',
-           'Steak', 'Meat Tornado', 'A Literal Garden\n']
-desserts = ['Desserts:', '----------', 'Ice Cream', 'Cake', 'Pie\n']
-drinks = ['Drinks', '----------', 'Coffee', 'Tea', 'Blood of the Innocent\n']
+menu = {
+    'appetizers' : ['Appetizers:\n','----------', 'Wings', 'Cookies', 'Spring Rolls\n'],
+'entrees' : ['Entrees:', '----------','Salmon',
+           'Steak', 'Meat Tornado', 'A Literal Garden\n'],
+'desserts' : ['Desserts:', '----------', 'Ice Cream', 'Cake', 'Pie\n'],
+'drinks' : ['Drinks', '----------', 'Coffee', 'Tea', 'Blood of the Innocent\n'],
+}
```

Comment:
You also try using a dictionary of dictionaries, might make things for the future of this code work a bit easier.